### PR TITLE
Max Stackhouse - Mod 2 - PD Deliverables

### PR DESCRIPTION
**Pull request by Maxscores**

### CODE REVIEW

File: deliverable_submissions/1710-b/max_stackhouse.md [hunk 1]
```diff
@@ -13,16 +13,18 @@ To make sure that you're in the right version of this markdown, check out [this
...
-  * How you've used agile as a process at Turing and what kind of project management tools you utilized. Address these questions:
-    * What were you already doing?
-    * What did you put into place in Module Two?
-    * What was effective?
-    * What do you want to improve on in future projects?
-  * Description of the role you played in your pair/group projects this module -- what was your role as a teammate? How did you utilize your strengths and leadership skills?
+  https://gist.github.com/Maxscores/2fe69f1a109591498277e4c35ac4f953
+  
 * Feedback II reflection: In-person feedback conversation following these [guidelines](https://github.com/turingschool/career-development-curriculum/blob/master/module_two/feedback_conversation_reflection_guidelines.md)
+https://gist.github.com/Maxscores/3af3fcc18f440754d4ea9f735da2e8c6
```
Comment (kaythensley) [inline deliverable_submissions/1710-b/max_stackhouse.md]: Continue to work on the coaching aspect. When you have a team that works well together, think about why that is - and if that wasn't the case with the person's net team, what difficulties might they encounter?

File: deliverable_submissions/1710-b/max_stackhouse.md [hunk 2]
```diff
@@ -13,16 +13,18 @@ To make sure that you're in the right version of this markdown, check out [this
...
+
 * Working draft of professional story (1-2 paragraphs)
+ https://gist.github.com/Maxscores/379978c928b9efa7d6ba0801819f4657
```
Comment (kaythensley) [inline deliverable_submissions/1710-b/max_stackhouse.md]: As you continue to work on your story, I'd like to hear more about where you want to go in the future. What technology or industry is calling to you? How do you want to use these new skills?

**Comment by kaythensley** [comment]
@Maxscores Your PD deliverables are good to go. Thanks!

File: deliverable_submissions/1710-b/max_stackhouse.md (hunk 2 above)
Comment (Maxscores) [reply deliverable_submissions/1710-b/max_stackhouse.md]: Thanks Kayt, That is what I'm thinking too. Need to figure that part out.

**Pull request merged**